app/services/remove_nccc_email_service.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+# rubocop:disable Layout/LineLength
+class RemoveNcccEmailService < ::WasteExemptionsEngine::BaseService
+
+  # https://rubular.com/r/LPHA79nxD5k2ff
```

Comment:
this is really handy. (Make sure it is cleared on rubular.com once this is merged!)

Reply:
... in which case the comment should be deleted ....